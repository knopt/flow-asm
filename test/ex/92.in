3
13
0.9883660707234831
0.2620931496318839 9.05928000517179 -9.15906725543896 
-7.201424669026004 3.294065647469367 -3.947995312900268 
-5.360095387455011 6.052072513925694 -8.726730258103863 
3.4149376515102166 -3.962808701829883 -6.27047906179701 
-6.208329109188773 -1.2992441636259784 7.576428987577405 
7.812200685457256 -7.156064917818248 1.9184960682636198 
-1.6941441196176257 -8.706854819157696 -9.666983992207347 
-6.5299144798139 -8.373626121077315 -5.4479669683101895 
-7.406689656258905 -0.08002297687357718 5.7271092957320455 
8.118204539932488 -6.882999197740245 8.096260882493421 
7.924229940870887 8.546830575470107 7.841795723847682 
6.304081608839013 -7.222440226156972 -2.0546743418397746 
5.922210196705082 -2.6522716547505043 -2.89551891885933 

3.1501521642776513 9.78461942857562 5.51681099841953 -6.520422694557777 0.469752387333326 8.205383055311952 -6.311581508307111 0.12492859947221113 -4.434473274670323 3.3063852097928574 0.7583534869219566 -1.46140409243303 -3.683146033706022 
