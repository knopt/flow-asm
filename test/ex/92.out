5.151606791433989 -21.405232048291264 26.306029777156002 
41.585020093685316 -10.205436087458528 16.07590318342569 
26.065001510708957 -33.557377794797425 29.620397038077613 
-25.422591582005722 12.259624152040896 24.36107456636792 
37.69834770378944 -0.9076965932110692 -46.43999636566954 
-36.26884968961424 18.229082851664337 -26.591120170284945 
9.940641275450426 18.564858510901864 10.662622045571071 
6.247299031036302 8.873860496770682 0.6200632601044278 
29.77423733362241 -20.514402566369334 -35.11603222439909 
-31.854498926862032 44.035197699379076 -16.937659073262562 
-14.408424224645337 -25.544955626959943 -30.433669647217002 
-15.499436542842595 54.21170815130496 11.675477029677648 
-10.491683094206428 10.157611410942923 -6.100621536363374 
